frontend/app/src/screens/diff/data-diff-node.tsx

```diff
@@ -201,34 +163,24 @@ export const DataDiffNode = (props: tDataDiffNodeProps) => {
       {commentsCount && (
         <div className="flex items-center" data-cy="comments-count" data-testid="comments-count">
           <Tooltip enabled content={"Total number of comments"}>
-            <div className="flex">
-              <ChatBubbleLeftRightIcon className="w-4 h-4 mr-2" />
-              <Pill className="mr-2">{JSON.stringify(commentsCount)}</Pill>
+            <div>
+              <Badge variant={"dark-gray"} className="rounded-full mr-2">
+                <Icon icon="mdi:message-fast-outline" className="mr-1" />
```

Comment:
I find it unclear at first to have 2 icons. After playing with it, I undertstood the 1st one is the count of comments and the 2nd is the count of thread. I feel like just having the count of comments is enough. WDYT?
<img width="885" alt="image" src="https://github.com/user-attachments/assets/b2c389e6-13df-4cbd-be73-b427eda35f14">

Reply:
the second one is for the total amount of comments accros the object; the first one is for the node only

it's not very clear but it's what we had previously, that's something to test and discuss once it's merged because it's not in the figma but it's important (for example if there is a comment on an attribute, we won't see it until we open the node)